src/group.spec.js

```diff
@@ -154,7 +150,52 @@ describe('group', () => {
       // delete the group
       await xo.call('user.delete', {id: userId})
       const group = await getGroup(groupId)
-      expect(group.users).to.be.a.permutationOf([])
+      expect(group.users).toEqual([])
+    })
+    /*
+      TO FIX: The key of the first user from the list of users has not been deleted from redis
+    */
```

Comment:
The message is incorrect: *some users are not properly deleted because of a race condition with group deletion*.